### PR TITLE
Incorporated PM Feedback

### PR DESCRIPTION
**Pull request by pabloang1**
Untethered vendor writer incorporating PM feedback.

### CODE REVIEW

**Comment by pabloang1** [comment]
@microsoft-github-policy-service agree company="Steyer Content"

File: microsoft-365/lighthouse/m365-lighthouse-understand-deployment-statuses.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Tasks can have the following statuses:
...
-|Not Compliant|<ul><li>One or more settings included in the subtask are either Missing or Not compliant.</li><li>There may be Extra settings detected within existing configurations.</li><p>**Note:** Doesn't apply to subtasks that aren't licensed. </p></ul>|
-|Not licensed|The tenant isn't licensed for the services required to deploy the configuration associated with the subtask.|
+|Compliant|<ul><li>All settings included in the subtask are Compliant.</li><li>There are no settings that are Not compliant.</li><li>There are no settings that are Missing from all existing configurations. A task can be Compliant if a setting is Compliant in one or more existing configurations without being Not compliant in another.</li><li>There may be Extra settings detected within existing configurations.</li></ul>|
+|Not compliant|<ul><li>One or more settings included in the subtask are Not compliant.</li><li>One or more settings are Missing from all existing configurations.</li><li>There may be Extra settings detected within existing configurations.</li><p>**Note:** Does not apply to subtasks that are not licensed. </p></ul>|
```
Comment (JohanFreelancer9) [inline microsoft-365/lighthouse/m365-lighthouse-understand-deployment-statuses.md]:
```suggestion
|Not compliant|<ul><li>One or more settings included in the subtask are Not compliant.</li><li>One or more settings are Missing from all existing configurations.</li><li>There may be Extra settings detected within existing configurations.</li><p>**NOTE:** Does not apply to subtasks that are not licensed. </p></ul>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pabloang1, kindly commit the minor suggested change for consistency. Many thanks.

**Comment by pabloang1** [comment]
Suggested change accepted..

**Comment by pabloang1** [comment]
@SKjerland, updates related to ADO 7669930, 7670113, 7669922

File: microsoft-365/lighthouse/m365-lighthouse-overview-deployment-task.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ A tenant's deployment plan is complete when all tasks are compliant or explicitl
 
 |Factor|Description|
 |---|---|
-|Is the tenant licensed for the required services associated with the task?|<ul><li>Tasks that aren't licensed won't be eligible for deployment.</li><li>Tasks that aren't licensed may be dismissed.</li><li>Lighthouse will direct users to the Business Workshop Tool to determine licensing needs.</li><li>Once Lighthouse detects that the tenant is licensed for the required services associated with the task, the status will be updated automatically.</li></ul>|
-|Is the task related to other tasks?|<ul><li>Tasks related to other tasks must be completed in sequential order.</li><li>Subsequent tasks will be eligible for deployment once the pre-requisite tasks are compliant.</li></ul>|
+|Is the tenant licensed for the required services associated with the task?|<ul><li>Tasks that are not licensed won't be eligible for deployment.</li><li>Tasks that aren't licensed may be Dismissed.</li><li>Lighthouse will direct users to the Business Workshop Tool to determine licensing needs.</li><li>Once Lighthouse detects that the tenant is licensed for the required services associated with the task, the status will be updated automatically.</li></ul>|
```
Comment (SKjerland) [inline microsoft-365/lighthouse/m365-lighthouse-overview-deployment-task.md:30]: @pabloang1 - Is there a reason why you removed the contraction from "aren't" in the first bulleted item but left the contraction in "aren't" in the second one? Microsoft voice is to use contractions, so they should both be contractions. At the very least, they should be the same for consistency.

File: microsoft-365/lighthouse/m365-lighthouse-understand-deployment-statuses.md [hunk 3]
```diff
@@ -31,9 +31,9 @@ Tasks can have the following statuses:
...
-|Not licensed|The tenant isn't licensed for the services required to deploy the configuration associated with the subtask.|
+|Compliant|<ul><li>All settings included in the subtask are Compliant.</li><li>There are no settings that are Not compliant.</li><li>There are no settings that are Missing from all existing configurations. A task can be Compliant if a setting is Compliant in one or more existing configurations without being Not compliant in another.</li><li>There may be Extra settings detected within existing configurations.</li></ul>|
+|Not compliant|<ul><li>One or more settings included in the subtask are Not compliant.</li><li>One or more settings are Missing from all existing configurations.</li><li>There may be Extra settings detected within existing configurations.</li><p>**NOTE:** Does not apply to subtasks that are not licensed. </p></ul>|
+|Not licensed|The tenant is not licensed for the services required to deploy the configuration associated with the subtask.|
```
Comment (SKjerland) [inline microsoft-365/lighthouse/m365-lighthouse-understand-deployment-statuses.md:36]: @pabloang1 - Shouldn't this be "aren't licensed"? Unless you're talking about a status, then it should be "Not licensed".

**Review by SKjerland — CHANGES_REQUESTED** [review]
@pabloang1 - I don't know why Kyle had you remove the contractions. That goes against Microsoft voice principles. He and I had a conversation about this just last week. I commented on a couple, but then noticed the contractions were removed all over the place. We need to put them back, unless there's a specific reason why he wants them removed. Also, if he's wanting to capitalize some of these things because he wants to refer to the setting itself and not the state, then you need to make those settings bold.

File: microsoft-365/lighthouse/m365-lighthouse-overview-deployment-task.md (hunk 2 above)
Comment (pabloang1) [reply microsoft-365/lighthouse/m365-lighthouse-overview-deployment-task.md:30]: Kyle's edits rolled back some contractions used. I'm now questioning if I should put them back.

File: microsoft-365/lighthouse/m365-lighthouse-understand-deployment-statuses.md (hunk 3 above)
Comment (pabloang1) [reply microsoft-365/lighthouse/m365-lighthouse-understand-deployment-statuses.md:36]: Same as previous, this a contraction rollback that I should put back.

**Comment by pabloang1** [comment]
> @pabloang1 - I don't know why Kyle had you remove the contractions. That goes against Microsoft voice principles. He and I had a conversation about this just last week. I commented on a couple, but then noticed the contractions were removed all over the place. We need to put them back, unless there's a specific reason why he wants them removed. Also, if he's wanting to capitalize some of these things because he wants to refer to the setting itself and not the state, then you need to make those settings bold.

**Comment by pabloang1** [comment]
I will update the content again and put contractions back and bold deployment status names.

**Pull request closed without merging**